src/components/Card.tsx

```diff
@@ -258,16 +258,18 @@ export const CardComponent: ComponentConfig<CardProps> = {
     image: {
       photo: {
         entityField: {
-          fieldName: "",
-          staticValue: "",
+          field: "",
+          constantValue: "",
+          constantValueEnabled: true
```

Comment:
Sorry, I should've been more clear -- if the constant value is blank, we want it to default to Select a Content field so leave constantValueEnabled off for these